index.js

```diff
@@ -11,7 +11,16 @@ class AggregateError extends Error {
 			throw new TypeError(`Expected input to be iterable, got ${typeof errors}`);
 		}
 
-		errors = Array.from(errors).map(err => err instanceof Error ? err : new Error(err));
+		errors = Array.from(errors).map(err => {
+			if (err instanceof Error) {
+				return err;
+			}
+			if (err && typeof err === 'object') {
```

Comment:
I think a `message` property should be a minimum, otherwise, the error is useless, as you got neither a correct stack nor a message. So if it's missing `message` it should throw a `TypeError`.

Reply:
Good point. I agree with that reasoning.